### PR TITLE
Commented options to disable cache for local development.

### PR DESCRIPTION
**Pull request by keopx**
Commented options to disable cache for local development.

This configuration is useful to develop locally so you don't have to constantly clear the cache (drush cr) and avoid cache with xdebug

### CODE REVIEW

File: web/sites/default/example.settings.local.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
  * Default configuration for local environments.
  */
 
-// Only setup database when env vars are available.
+/** Only setup database when env vars are available. */
```
Comment (dlopez-akalam) [inline web/sites/default/example.settings.local.php]:
```suggestion
// Only setup database when env vars are available.
```

File: web/sites/default/example.settings.local.php [hunk 2]
```diff
@@ -19,5 +19,43 @@
   ];
 }
 
+/** Config split environment. */
```
Comment (dlopez-akalam) [inline web/sites/default/example.settings.local.php]:
```suggestion
// Config splits.
```

File: web/sites/default/example.settings.local.php [hunk 3]
```diff
@@ -19,5 +19,43 @@
...
+//  $settings['cache']['bins']['discovery_migration'] = 'cache.backend.memory';
+//
+//  /** Disable Internal Page Cache. */
+//  $settings['cache']['bins']['page'] = 'cache.backend.null';
+//
+//  /** Disable Dynamic Page Cache. */
+//  $settings['cache']['bins']['dynamic_page_cache'] = 'cache.backend.null';
+//
+//  /** Skip file system permissions hardening. */
+//  $settings['skip_permissions_hardening'] = TRUE;
```
Comment (dlopez-akalam) [inline web/sites/default/example.settings.local.php]:
```suggestion
// Assertions.
// assert_options(ASSERT_ACTIVE, TRUE);
// \Drupal\Component\Assertion\Handle::register();

// Enable local development services.
// $settings['container_yamls'][] = DRUPAL_ROOT . '/sites/development.services.yml';

// Show all error messages, with backtrace information.
// $config['system.logging']['error_level'] = 'verbose';

// Disable CSS and JS aggregation.
// $config['system.performance']['css']['preprocess'] = FALSE;
// $config['system.performance']['js']['preprocess'] = FALSE;
// Disable the render cache.
// $settings['cache']['bins']['render'] = 'cache.backend.null';

// Disable caching for migrations.
// $settings['cache']['bins']['discovery_migration'] = 'cache.backend.memory';

// Disable Internal Page Cache.
// $settings['cache']['bins']['page'] = 'cache.backend.null';

// Disable Dynamic Page Cache.
// $settings['cache']['bins']['dynamic_page_cache'] = 'cache.backend.null';

// Skip file system permissions hardening.
// $settings['skip_permissions_hardening'] = TRUE;

```

**Comment by dlopez-akalam** [comment]
@keopx thanks for contributing, I have some suggestions related to the format, can you please review it?

**Comment by dlopez-akalam** [comment]
Please note that the latest suggestion is wrongly removing only the last line, I don't know how to do it better but I think the idea is clear, please let me know if you have any doubt.

**Comment by keopx** [comment]
@dlopez-akalam I updated commit. Thanks for you review.

**Review by dlopez-akalam — APPROVED** [review]

**Comment by dlopez-akalam** [comment]
Merged! thanks for contributing.

**Pull request merged**